client/app/visualizations/chart/index.js

```diff
@@ -87,7 +88,12 @@ function ChartEditor(ColorPalette, clientConfig) {
       };
       scope.chartTypeChanged();
 
+      scope.colorScheme = ['Blackbody', 'Bluered', 'Blues', 'Earth', 'Electric',
+        'Greens', 'Greys', 'Hot', 'Jet', 'Picnic', 'Portland',
+        'Rainbow', 'RdBu', 'Reds', 'Viridis', 'YlGnBu', 'YlOrRd', 'Custom..'];
```

Comment:
`'Custom..'` -> `'Custom...'` (thee dots) + fix value in `ng-if`s in editor template and js code